src/js/components/Fittable.jsx

```diff
@@ -439,14 +444,24 @@ class Fittable extends React.Component {
     this.hammer.destroy()
   }
 
+  onWindowResize (e) {
+    const mobile = this.isSmallScreen()
+    if (this.state.isMobile !== mobile) {
+      this.setState({ isMobile: mobile })
+    }
+  }
+
   componentDidMount () {
     this.registerSwipeListener(this.refs.rootEl.getDOMNode())
     this.refs.rootEl.getDOMNode().addEventListener('click', this.handleClick.bind(this))
+    window.addEventListener('resize', this.onWindowResize.bind(this))
+    window.addEventListener('load', this.onWindowResize.bind(this))
   }
 
   componentWillUnmount () {
     this.unregisterSwipeListener()
     this.refs.rootEl.getDOMNode().removeEventListener('click', this.handleClick.bind(this))
+    window.removeEventListener('resize', this.onWindowResize.bind(this))
```

Comment:
Since Fittable component doesn't get unmounted anytime, I don't think this is a big issue. We can solve it throughout refactoring, I'd rather prefer to merge this PR already.